user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md

```diff
@@ -0,0 +1,287 @@
+---
+uid: DOM_Making_DOM_Stateful
+---
+
+# Making DOM objects state-aware
+
+This tutorial shows how you can make your DOM objects state-aware. This allows a DOM instance to behave differently depending on the state it transitions to. The following things can be defined for each state:
+
+- Which fields are **visible** to the user.
+- Which fields are **read-only** for the user.
+- Which fields are **mandatory**.
+- To which state the instance can **transition**.
+- What **actions** are allowed.
+- What **buttons** to show on the form.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.3.
+
+Expected duration: 20 minutes
+
+<!-- Uncomment this section when the link to the Kata is available:
+> [!TIP]
+> This tutorial is covered in the following Kata : [Kata #24: Making DOM objects state-aware](https://community.dataminer.services/courses/kata-24/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
+>
+> If you are new to DataMiner Object Models (DOM), first watch [Kata #15: Getting started with DataMiner Object Models](https://community.dataminer.services/courses/kata-15/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png) -->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.10 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- An [indexing database](xref:Indexing_Database) or [Storage as a Service](xref:STaaS).
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
```

Comment:
Note that this overview should only contain the steps below the overview, not the prerequisites etc. above it.